### PR TITLE
Support Amazon Linux 2

### PR DESCRIPTION
**Pull request by hayajo**

### CODE REVIEW

File: tasks/amazonlinux.yml [hunk 1]
```diff
@@ -1,9 +1,23 @@
...
+  shell: yum info installed system-release 2>/dev/null | awk '/^Version/ { print $3 }'
+  register: release_version
+
 - name: import mackerel GPG key
   rpm_key: key=https://mackerel.io/file/cert/GPG-KEY-mackerel
+  when: release_version.rc == 0 and release_version.stdout|float < 2017.12
+
+- name: import mackerel GPG key v2
+  rpm_key: key=https://mackerel.io/file/cert/GPG-KEY-mackerel-v2
+  when: release_version.rc == 0 and release_version.stdout|float >= 2017.12
```
Comment (astj) [inline tasks/amazonlinux.yml]: Hmm. 

https://aws.amazon.com/jp/amazon-linux-2/faqs/

> With the availability of Amazon Linux 2, we are announcing that 2017.09 AMI release for Amazon Linux and container image will be the last release for the current generation of Amazon Linux. Going forward, AWS will provide newer versions only for Amazon Linux 2.

File: tasks/amazonlinux.yml [hunk 2]
```diff
@@ -1,9 +1,23 @@
 ---
+- name: register release version
+  shell: yum info installed system-release 2>/dev/null | awk '/^Version/ { print $3 }'
```
Comment (astj) [inline tasks/amazonlinux.yml]: I don't think this is the good solution, but I don't have a good alternative……
(Ansible should set proper `distribution_version` for AL2...)

File: tasks/amazonlinux.yml [hunk 3]
```diff
@@ -1,9 +1,23 @@
...
+  register: release_version
+
 - name: import mackerel GPG key
   rpm_key: key=https://mackerel.io/file/cert/GPG-KEY-mackerel
+  when: release_version.rc == 0 and release_version.stdout|float < 2017.12
```
[the remark this answers is not in the document]
Comment (hayajo) [reply tasks/amazonlinux.yml]: When `release_version.rc != 0`, it should end and the remaining tasks should not be executed. 
But `release_version.rc` does not become 0.

Change to check the `release_version.stdout`.

File: tasks/amazonlinux.yml [hunk 4]
```diff
@@ -1,9 +1,27 @@
...
   rpm_key: key=https://mackerel.io/file/cert/GPG-KEY-mackerel
+  when: release_version.stdout|float < 2017.12
+
+- name: import mackerel GPG key v2
+  rpm_key: key=https://mackerel.io/file/cert/GPG-KEY-mackerel-v2
+  when: release_version.stdout|float >= 2017.12
 
 - name: add repository 'mackerel'
   copy: src=amazonlinux-mackerel.repo dest=/etc/yum.repos.d/mackerel.repo owner=root group=root mode=0644
+  when: release_version.rc == 0 and release_version.stdout|float < 2017.12
```
Comment (astj) [inline tasks/amazonlinux.yml]: `release_version.rc` still exists here. (and L24)

Comment (hayajo) [reply tasks/amazonlinux.yml]: Sorry. Fixed.

**Pull request merged**